ios/GoldWallet.xcodeproj/project.pbxproj

```diff
@@ -944,7 +960,7 @@
 					"$(SRCROOT)/../node_modules/react-native-splash-screen/ios",
 				);
 				INFOPLIST_FILE = GoldWallet/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
should stay 9.0, check all IPHONEOS_DEPLOYMENT_TARGET modified here, they should all be 9.0

Reply:
done